packages/core/lib/commands/commands.js

```diff
@@ -0,0 +1,27 @@
+module.exports = [
+  "build",
+  "compile",
+  "config",
+  "console",
+  "create",
+  "dashboard",
+  "db",
+  "debug",
+  "deploy",
+  "develop",
+  "exec",
+  "help",
+  "init",
+  "install",
+  "migrate",
+  "networks",
+  "obtain",
+  "opcode",
+  "preserve",
+  "publish",
+  "run",
+  "test",
+  "unbox",
+  "version",
+  "watch"
+];
```

Comment:
is it okay that this is now static?

Reply:
One thought I had is that we could generate it at build time - that might be a little strange though.